Update Example_C2_Profile Base Image

ulricvbs opened the pull request:
Update version of base python38_sanic_c2profile image to 0.0.6 and fix import in C2_RPC_functions.py.

Closes #237

its-a-feature commented:
This is a great idea! I think we should go one step further and actually include examples of the full suite of available RPC functions instead of just `opsec` and the `test` function. We can use the `http` profile as an example or the `dynamichttp` profile to show the configuration check and redirector rule generation functions

ulricvbs commented:
Sounds good, I'll work on adding these in with documentation since I'm going through the process of building a C2 profile.

ulricvbs commented:
Ok, I think everything is done; added to the pull request. Let me know if I'm missing something. I added docstrings/comments to explain the function's purpose.
